rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/AbstractJobTagPaginatedCommand.java

```diff
@@ -0,0 +1,28 @@
+package org.ow2.proactive_grid_cloud_portal.cli.cmd.sched;
+
+public abstract class AbstractJobTagPaginatedCommand extends AbstractJobTagCommand {
+
+    protected final int MAX_PAGE_SIZE = 50;
```

Comment:
MAX_PAGE_SIZE parameter must not be hardcoded and configurable in config/web/settings.ini